docs/collections.md

```diff
@@ -0,0 +1,138 @@
+# Collections
+Collections define the structure for the different content types on your static site. Since every site is different, the `collections` settings will be very different from one site to the next.
+
+Let's say your site has a blog, with the posts stored in `_posts/blog`, and files saved in a date-title format, like `1999-12-31-lets-party.md`. Each post begins with settings in yaml-formatted front matter, like so:
```

Comment:
Mix in a little more explanation that, based on the scenario, it would be best to use a folder type collection.

Also, "settings in YAML front matter".